plugin.program.AML/changelog.txt

```diff
@@ -18,6 +18,12 @@ WIP      [CORE] AML/AEL must report the aspect ratio of artwork to the skin.
 
 [B]AML ideas and planned features (read to implement)[/B]
 
+WIP      [CORE] Implement the "Read-only launchers". ROLs are XML files that include the
```

Comment:
this file is not needed in submissions, it'd be nice if you could drop it in the future. It is not parsed by kodi anymore in favor of `<news>`

Reply:
I know. However, AML is a very atypical addon due to its huge size and many features as an emulator frontend. Most of the times the `<news>` tag is insufficient to convey all the information of the new version. I think the best way is to include the `changelog` file and refer to it in the `<news>` tag.